### PR TITLE
Refactor CSS variables to remove changeOpacity

### PR DESCRIPTION
**Pull request by bradjcoughlin**
#### Summary
This removes the variations of the default theme variables utilizing `changeOpacity()` in javascript and implements the CSS `rgba()` function to apply opacity mutations downstream. The goal is to reduce the number of initial global variables and allow the CSS to handle any variations using pure CSS.

#### Ticket Link
N/A

#### Related Pull Requests
Upstream initial changes began in https://github.com/mattermost/mattermost-webapp/pull/5622, so this PR needs to wait until it is merged before being merged.

### CODE REVIEW

File: sass/layout/_sidebar-left.scss [hunk 1]
```diff
@@ -228,7 +228,7 @@
 
             i {
                 font-size: 16px;
-                color: var(--rgba(var(--center-channel-color-rgb), 0.72));
```
Comment (bradjcoughlin) [inline sass/layout/_sidebar-left.scss]: typo on previous commit

File: components/widgets/modals/full_screen_modal.scss [hunk 2]
```diff
@@ -27,7 +27,7 @@
             border-radius: 4px;
 
             &:hover {
-                fill: var(--rgba(var(--center-channel-color-rgb), 0.72));
```
Comment (bradjcoughlin) [inline components/widgets/modals/full_screen_modal.scss]: typo on previous commit

File: components/widgets/modals/full_screen_modal.scss [hunk 3]
```diff
@@ -27,7 +27,7 @@
...
+                fill: rgba(var(--center-channel-color-rgb), var(--alpha-900));
                 background: rgba(var(--center-channel-color-rgb), 0.08);
```
Comment (bradjcoughlin) [inline components/widgets/modals/full_screen_modal.scss:31]:
```suggestion
                background: rgba(var(--center-channel-color-rgb), var(--alpha-200));
```

**Comment by jasonblais** [comment]
Hey @bradjcoughlin, hope you are doing well :) Just touching base to see if you're still interested working on this PR? I noticed it's in draft stage at the moment.

**Comment by hanzei** [comment]
Closing a the ticket is not longer valid.

**Pull request closed without merging**